import "mapreduce/yt/interface/protos/extension.proto";

message TMapperOutputRow {
    optional uint64 Index = 1 [(NYT.column_name) = "idx"];
    optional string Subnet = 2 [(NYT.column_name) = "subnet"];
}
